### ISSUE TITLE
WARNING: An illegal reflective access operation has occurred

### ISSUE THREAD

**Issue by christophfink** [r5py/r5py#263]
R5 uses a deprecated version of `org.mapdb`, which results in warnings about illegal reflective access.
We tried to suppress the warnings on the Python console and especially in notebook output (see #26), but did not manage at that time. 

Let’s revisit this

**Comment by wklumpen** [comment]
Is there an upstream issue open on this in R5? Should we open one?

**Comment by christophfink** [comment]
Good point, I think there was one at some point. At the very least, in the source code, they make a compelling argument for sticking to the old version of mapdb, if I remember correctly.

Here: https://github.com/conveyal/r5/blob/8545de9f2ee12f240e910d42a944c96e57459348/build.gradle#L186

**Comment by christophfink** [comment]
So, it might be worth filing a bug upstream (or commenting on a possibly already opened report), but our best chances might be still to somehow capture the JVM’s output (weirdly, it should (MUST!) be stderr or stdout, but I cannot seem to tap on to it)
